src/zcash-proving-service.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "streams.h"
+#include "util.h"
+#include "version.h"
+#include "zcash/JoinSplit.hpp"
+#include "zcash/Proof.hpp"
+
+#include <vector>
+#include <zmq.hpp>
```

Comment:
In my environment `zmq.hpp` is not present and build fails. I hacked it by downloading `https://github.com/zeromq/cppzmq/blob/master/zmq_addon.hpp` into `depends/x86_64-unknown-linux-gnu/include` and succeeded. 

I might did something wrong, but because you don't have automated builds I am not sure if your PR builds, so this might be worth looking into.